src/PluginsSystem/Microsoft.Performance.Toolkit.Plugins.Runtime/Installation/Storage/DefaultPluginsStorageDirectory.cs

```diff
@@ -37,16 +36,25 @@ public string GetPluginContentDirectory(PluginIdentity pluginIdentity)
             Guard.NotNull(pluginIdentity, nameof(pluginIdentity));
 
             string directory = GetPluginRootDirectory(pluginIdentity);
-            return Path.GetFullPath(Path.Combine(directory, pluginContentFolder));
+            return Path.GetFullPath(Path.Combine(directory, PackageConstants.PluginContentFolderName));
```

Comment:
I'd rather define the file name are folder name in this class although they are the same as the package constants right now. They define where the files should be places in this particular storage implementation whereas the package constants define the structure of the package.